packages/react-components/src/helpers/options-for-select.js

```diff
@@ -1,128 +0,0 @@
-import i18next from 'i18next';
-
-export const months = () => [
-  { label: i18next.t('january'), value: 1 },
-  { label: i18next.t('february'), value: 2 },
-  { label: i18next.t('march'), value: 3 },
-  { label: i18next.t('april'), value: 4 },
-  { label: i18next.t('may'), value: 5 },
-  { label: i18next.t('june'), value: 6 },
-  { label: i18next.t('july'), value: 7 },
-  { label: i18next.t('august'), value: 8 },
-  { label: i18next.t('september'), value: 9 },
-  { label: i18next.t('october'), value: 10 },
-  { label: i18next.t('november'), value: 11 },
-  { label: i18next.t('december'), value: 12 },
-];
```

Comment:
We'll need to use this once we translate our date web components.

https://github.com/department-of-veterans-affairs/component-library/blob/2207aea1a98a7fe114e093ba720f853ded8d74ad/packages/web-components/src/utils/date-utils.ts#L7-L20